src/js/tabs/firmware_flasher.js

```diff
@@ -575,10 +575,10 @@ firmware_flasher.initialize = function (callback) {
         }
 
         let result = getConfig('erase_chip');
-        if (result.erase_chip) {
+        if (result.erase_chip === undefined) {
             $('input.erase_chip').prop('checked', true);
         } else {
-            $('input.erase_chip').prop('checked', false);
+            $('input.erase_chip').prop('checked', result.erase_chip);
         }
```

Comment:
I think is better to "overcharge" the getConfig() method with this code, adding a second "optional" parameter as "default". If configured, and undefined, then return the default. In this way we can do the same for others parameters if we want. What do you think?

Reply:
Agree, its a better option, but then we have to change them all.

Reply:
Seems good to me 😉 thanks!